content/100.figure-table-legends.md

```diff
@@ -14,10 +14,11 @@ For example, 75 single-cell samples and 101 single-nuclei samples have accompany
 Two samples were sequenced using both single-cell and single-nuclei suspensions so are included in the count for both "Single-cell" and "Single-nuclei" groups.
 Samples that were sequenced with either bulk RNA-seq or spatial transcriptomics and do not have accompanying single-cell or single-nuclei RNA-seq data are not represented in the total counts.
 
-C. Example of a project card as displayed on the "Browse" page of the ScPCA Portal.
-This project card is associated with project `SCPCP000009` [@doi:10.21203/rs.3.rs-2517703/v1; @doi:10.21203/rs.3.rs-2517758/v1].
+C. Example of a project card as displayed on the "Browse" page of the ScPCA Portal and a "Visualize" view for a library within that project, colored by consensus cell type annotation..
+This project card and visualized sample are from project `SCPCP000004` [@doi:10.1101/2024.01.07.574538; @doi:10.1186/s13059-024-03309-4].
 Project cards include information about the number of samples, technologies and modalities, additional sample metadata information, submitter-provided diagnoses, and a submitter-provided abstract.
 Where available, submitter-provided citation information, as well as other databases where this data has been deposited, are also provided.
+The visualization employs the UCSC Cell Browser [@doi:10.1093/bioinformatics/btab503], which allows interactive exploration of the UMAP embeddings for each cell, with user-selectable coloring options that include cell types, gene-level expression, and other per-cell annotations created by the `scpca-nf` workflow.
```

Comment:
Just noting the dictionary sadly does not like `selectable`, so we might want to add that in.